_data/Spring21/JoseSebastian.json

```
@@ -0,0 +1,13 @@
+{
+  "emoji": ":smirk_cat:",
+  "role": "Student",
+  "introduction": "I'm Jose Sebastian, I'm 19 years old and I'm really excited to learn more about programming this semester!",
+  "achievements": [
+    "I won a mini videogames tournament",
+    "I was able to build mini programs in Java."
```

```suggestion
    "I was able to build mini programs in Java.",
```
Don't miss the commas in JSON files